turbobt/substrate/pallets/author.py

```diff
@@ -10,14 +10,20 @@
 from .state import RuntimeVersion
 
 
+class Era(typing.TypedDict):
+    period: int
```

Comment:
Is it okay to not have any explanation of these values here? I don't understand them